packages/contracts/deploy/update/to_v1.3.0/20_TokenVoting_Plugin.ts

```diff
@@ -0,0 +1,88 @@
+import {DeployFunction} from 'hardhat-deploy/types';
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {PluginRepo__factory} from '../../../typechain';
+import {getContractAddress, uploadToIPFS} from '../../helpers';
+
+import tokenVotingSetupArtifact from '../../../artifacts/src/plugins/governance/majority-voting/token/TokenVotingSetup.sol/TokenVotingSetup.json';
+import tokenVotingReleaseMetadata from '../../../src/plugins/governance/majority-voting/token/release-metadata.json';
+import tokenVotingBuildMetadata from '../../../src/plugins/governance/majority-voting/token/build-metadata.json';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  console.log('\nUpdate TokenVoting Plugin');
+  const {deployments, ethers, network} = hre;
+  const {deploy} = deployments;
+  const [deployer] = await ethers.getSigners();
+
+  const deployResult = await deploy('TokenVotingSetup', {
+    contract: tokenVotingSetupArtifact,
+    from: deployer.address,
+    args: [],
+    log: true,
+  });
+
+  const tokenVotingReleaseCIDPath = await uploadToIPFS(
+    JSON.stringify(tokenVotingReleaseMetadata),
+    network.name
+  );
+  const tokenVotingBuildCIDPath = await uploadToIPFS(
+    JSON.stringify(tokenVotingBuildMetadata),
+    network.name
+  );
+
+  const tokenVotingRepoAddress = await getContractAddress(
+    'token-voting-repo',
+    hre
+  );
+  const tokenVotingRepo = PluginRepo__factory.connect(
+    tokenVotingRepoAddress,
+    ethers.provider
+  );
+  if (
+    await tokenVotingRepo.callStatic.isGranted(
+      tokenVotingRepoAddress,
+      deployer.address,
+      await tokenVotingRepo.MAINTAINER_PERMISSION_ID(),
+      '0x00'
+    )
+  ) {
+    console.log(`Deployer has permission to install new TokenVoting version`);
+    const tx = await tokenVotingRepo
+      .connect(deployer)
+      .createVersion(
+        1,
```

Comment:
Very minor, but I would create a constant like `TARGET_RELEASE` at the top, factoring out all occurrences of this

Reply:
Added in bd20b893734d9afa715bf68128e4514ffeed67ea